### PR TITLE
Modify to use library-go manifest and verify packages

### PR DESCRIPTION
**Pull request by jottofar**
library-go manifest and verify packages were created since both CVO and the oc client use them.

### CODE REVIEW

File: pkg/cvo/cvo.go [hunk 1]
```diff
@@ -691,14 +691,14 @@ func (b *resourceBuilder) builderFor(m *lib.Manifest, state payload.State) (reso
 	if resourcebuilder.Mapper.Exists(m.GVK) {
 		return resourcebuilder.New(resourcebuilder.Mapper, config, *m)
 	}
-	client, err := dynamicclient.New(config, m.GVK, m.Object().GetNamespace())
+	client, err := dynamicclient.New(config, m.GVK, m.Obj.GetNamespace())
```
Comment (wking) [inline pkg/cvo/cvo.go:694]: I don't understand this change.  Did we loose the getter somewhere?  Although not clear to me why we'd have a getter in front of a public property...

Comment (jottofar) [reply pkg/cvo/cvo.go:694]: Comment from when I merged `manifest` into library-go. The method isn't needed: https://github.com/openshift/library-go/pull/680#discussion_r385556651

File: pkg/cvo/cvo_test.go [hunk 2]
```diff
@@ -39,10 +39,10 @@ import (
 	clientset "github.com/openshift/client-go/config/clientset/versioned"
 	"github.com/openshift/client-go/config/clientset/versioned/fake"
 
-	"github.com/openshift/cluster-version-operator/lib"
 	"github.com/openshift/cluster-version-operator/pkg/payload"
-	"github.com/openshift/cluster-version-operator/pkg/verify"
-	"github.com/openshift/cluster-version-operator/pkg/verify/store/sigstore"
+	"github.com/openshift/library-go/pkg/manifest"
+	//"github.com/openshift/library-go/pkg/verify"
```
Comment (wking) [inline pkg/cvo/cvo_test.go:44]: nit: drop this commented-out import?

Comment (jottofar) [reply pkg/cvo/cvo_test.go]: Done

**Comment by jottofar** [comment]
retest

**Comment by wking** [comment]
Integration is still blocked on openshift/release#13745 or similar.

/override ci/prow/integration

/lgtm

**Pull request merged**